solver/llbsolver/ops/source.go

```diff
@@ -67,21 +67,27 @@ func (s *sourceOp) CacheMap(ctx context.Context, g session.Group, index int) (*s
 	if err != nil {
 		return nil, false, err
 	}
-	k, cacheOpts, done, err := src.CacheKey(ctx, g, index)
+
+	k, srcDigest, cacheOpts, done, err := src.CacheKey(ctx, g, index)
 	if err != nil {
 		return nil, false, err
 	}
 
 	dgst := digest.FromBytes([]byte(sourceCacheType + ":" + k))
-
 	if strings.HasPrefix(k, "session:") {
 		dgst = digest.Digest("random:" + strings.TrimPrefix(dgst.String(), dgst.Algorithm().String()+":"))
 	}
 
+	var buildInfo map[string]string
+	if !strings.HasPrefix(s.op.Source.GetIdentifier(), "local://") {
+		buildInfo = map[string]string{s.op.Source.GetIdentifier(): srcDigest}
```

Comment:
I wonder if this can contain git/http credentials. If yes then they need to be removed. Eg. git does remove them on the output it produces.

Reply:
If token is directly added to the clone URL `https://<token>@github.com/owner/repo.git` then yes it will be displayed but not if we use the `GIT_AUTH_TOKEN` secret.

Reply:
Need to mask the inline url as well.